recipes/sqlite3/all/conanfile.py

```diff
@@ -172,12 +172,12 @@ def package_info(self):
         self.cpp_info.filenames["cmake_find_package"] = "SQLite3"
         self.cpp_info.filenames["cmake_find_package_multi"] = "SQLite3"
         self.cpp_info.set_property("cmake_file_name", "SQLite3")
+        self.cpp_info.set_property("cmake_target_name", "SQLite::SQLite3")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_target_name", "SQLite::SQLite3")
        self.cpp_info.set_property("cmake_find_mode", "both") 
```

https://cmake.org/cmake/help/latest/module/FindSQLite3.html